dolphinscheduler-alert/src/main/java/org/apache/dolphinscheduler/alert/utils/MailUtils.java

```diff
@@ -85,15 +101,6 @@
      */
     public static Map<String,Object> sendMails(Collection<String> receivers, Collection<String> receiversCc, String title, String content, String showType) {
         Map<String,Object> retMap = new HashMap<>();
-
-        // if mail is default config, no need to process
-        if (StringUtils.isEmpty(MAIL_SERVER_HOST) || "xxx.xxx.com".equals(MAIL_SERVER_HOST)) {
-            retMap.put(Constants.MAIL_ENABLED, false);
-            retMap.put(Constants.STATUS, true);
-            return retMap;
-        }
-
```

Comment:
I think the legality of the parameters that should be sent here needs to be judged. Btw, tools should not include business, therefore, do not use judgments such as xxx.xxx.com.

Reply:
Because the 1.3.5 release version needed a quick fix, I made this modification, although I know it is unreasonable. Now I have removed it.